### ISSUE TITLE
Feature Request: Automatic slice progressing during segmenting

### ISSUE THREAD

**Issue by gdevenyi** [BIC-MNI/Display#21]
From a segmenter in my group:
> Slice Slide progression.  Neuromorphometrics Inc. has produced a new tracing program that allows for faster tracing by implementing a simple button that progresses the slice everytime a drawing event occurs.  The use would be you have a cortex that you traced and now you must separate the cortex by drawing on the same anatomical point of every slice.  once you draw the line separating, it automatically progresses forward by one slice.

**Comment by rdvincent** [comment]
How is this better/different from using the keyboard '+' or '-' keys to switch slices (as is already possible in Display)? Does this really happen automatically after the drawing event, e.g after every "brush stroke"?

**Comment by gdevenyi** [comment]
Question relayed to user.

**Comment by gdevenyi** [comment]
Response:

> Yes this would be after the brush stroke occurs... Would just eliminate manually switching to the next slice

**Comment by rdvincent** [comment]
OK, that is quite simple to implement, but it obviously could not be the default.

**Comment by gdevenyi** [comment]
Definitely, just a switch somewhere to turn it on for certain operations.
